common/shlibs

```diff
@@ -3977,3 +3977,4 @@ libselinux.so.1 libselinux-3.0_1
 libsepol.so.1 libsepol-3.0_1
 libfrrcares.so.0 libfrr-7.3.1_1
 libhugetlbfs.so.0 libhugetlbfs-2.22_1
+libmonocypher.so.3 monocypher-3.1.1_1
```

Comment:
Any reason to add this, if nothing is using it?

Reply:
It offers an shared lib, so  Ι add it to common/shlibs. It is not the only entry in there that is not used by any other programs in the repo...

Reply:
fair enough